src/azul/service/responseobjects/step_function_helper.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from azul.deployment import aws
+
+
+class StepFunctionHelper:
```

Comment:
This wrapper is annoyingly anemic. It doesn't do much but requires a lot of documentation. I realize that you need this for the mocks but we should be true to ourselves by not inflating this class' importance with fluffy documentation.